packages/libs/components/src/map/BoundsDriftMarker.tsx

```diff
@@ -280,26 +280,28 @@ export default function BoundsDriftMarker({
 
   // add click events for highlighting markers
   const handleClick = (e: LeafletMouseEvent) => {
-    // if selected, unselect
-    if (setSelectedMarkers) {
-      if (selectedMarkers?.find((id) => id === props.id)) {
-        setSelectedMarkers((prevSelectedMarkers: string[]) =>
-          prevSelectedMarkers.filter((id: string) => id !== props.id)
-        );
-      } else {
-        // select
-        setSelectedMarkers((prevSelectedMarkers: string[]) => [
-          ...prevSelectedMarkers,
-          props.id,
-        ]);
+    // check the number of mouse click and enable function for single click only
+    if (e.originalEvent.detail === 1) {
```

Comment:
This is great. It seemed to me that it might not be a robust solution but `originalEvent.detail` is core Leaflet so it should be fine. No need to check for browser compatibility etc, which I suspected at first (from my first naive Google results).

Reply:
@bobular thank you for your review 👍 I have tested both firefox and chrome browsers and they worked fine. I will merge this into the parent branch